examples/private-endpoint/main.tf

```diff
@@ -17,6 +17,7 @@ provider "azurerm" {
 }
 
 # We need the tenant id.
+# tflint-ignore: terraform_unused_declarations
```

Comment:
As per [TFNFR36](https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tfnfr36---category-code-style---example-code-must-set-prevent_deletion_if_contains_resources-to-false-in-provider-block) set `prevent_deletion_if_contains_resources` in `providers`:
```terraform
resource_group {
      prevent_deletion_if_contains_resources = true
    }
```

Reply:
Updated as requested although this should be false